aesm-client/build.rs

```diff
@@ -18,6 +19,9 @@ fn main() {
         out_dir: out_dir.to_str().expect("OUT_DIR must be valid UTF-8"),
         input: &["src/aesm_proto.proto"],
         includes: &[],
+        customize: Customize {
```

Comment:
just write `customize: Default::default()`

Reply:
Thanks. Fixed in current push.